template/plop/reactComponent/templates/__tests__/component.spec.hbs

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { makeContextRenderer } from 'utils/testUtils';
 
+import { makePropsRenderer } from '{{ testUtilsPath }}';
```

Comment:
Now you have duplicated import - `makeContextRenderer` is already imported above